packages/client/components/DashNavList/DashNavList.tsx

```diff
@@ -56,44 +42,42 @@ const DashNavList = (props: Props) => {
     `,
     organizationsRef
   )
-  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
-  const teams = organizations?.flatMap((org) => org.viewerTeams)
+
+  const tiers = ['enterprise', 'team', 'starter']
```

Comment:
-1 we should use `TierEnum` here instead as the we might change the names, e.g. `team` back to `pro`

Reply:
I see this in other places, but I’m not sure the best way to import and define a `TierEnum` here.